### PR TITLE
file.lua copy for windows

### PR DESCRIPTION
**Pull request by supertobi**
Could be improved, but at least it works for me.

### CODE REVIEW

**Comment by supertobi** [comment]
@wpferguson Please test and merge if OK.

**Comment by wpferguson** [comment]
I'll build a test suite for this and the other PR.  Should have it done tonight.

File: lib/dtutils/file.lua [hunk 1]
```diff
@@ -291,9 +291,13 @@ dtutils_file.libdoc.functions["file_copy"] = {
 function dtutils_file.file_copy(fromFile, toFile)
   local result = nil
   -- if cp exists, use it
-  if dtutils_file.check_if_bin_exists("cp") then
+  if dt.configuration.running_os == "windows" then
+    os.execute('copy "' .. fromFile .. '" "' .. toFile .. '"')
+    result = true
```
Comment (wpferguson) [inline lib/dtutils/file.lua]: this should be result = os.execute also.  It returns nil on failure and true on success, so we can use the windows copy command and if it fails for some reason then fall back to the pure lua copy solution.

Comment (supertobi) [reply lib/dtutils/file.lua]: done

**Pull request merged**